infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/send/SnapshotSender.java

```diff
@@ -133,15 +136,17 @@ public void transmit(UUID snapshotSyncEventId) {
                     cancel = true;
                     break;
                 } catch (ReplicationReaderException e) {
-                   if (e.getCause() instanceof TimeoutException &&
-                       fsm.getSession().getSubscriber().getModel() == LogReplication.ReplicationModel.ROUTING_QUEUES) {
-                       log.info("Snapshot sync timed out waiting for data.  Will request a new snapshot sync");
-                       snapshotSyncCancel(snapshotSyncEventId, LogReplicationError.UNKNOWN, true);
-                   } else {
```

Comment:
by removing this, we will not handle ReplicationReaderException thrown from other models.

Reply:
We actually don't have any other model throwing ReplicationReaderException....but retaining the else part for future use.